src/code/engine_library/plotReportTimeProfile.m

```diff
@@ -83,17 +83,25 @@
     end
     
     if ~isempty(DataTP)
+        ylimD = inf;
         for iInd = 1:length(DataTP)
             DataTP(iInd).y = log10(DataTP(iInd).y);
             DataTP(iInd).lloq = log10(DataTP(iInd).lloq);
             jj = ~isinf(DataTP(iInd).y);
-            ylim = min([ylim;DataTP(iInd).y(jj)]);
+            ylimD = min([ylimD;DataTP(iInd).y(jj)]);
+        end
+        
+        if ylim < ylimD/3
+            ylim = ylimD/3;
+        else
+            ylim = min(ylim,ylimD);
         end
     end
     
     ylim = ylim*0.8;
-else
```

Comment:
Could you put some comments explaining this piece of code (especially what is the meaning of 1/3 and 0.8:

```
        if ylim < ylimD/3
            ylim = ylimD/3;
        else
            ylim = min(ylim,ylimD);
        end
    end
    
    ylim = ylim*0.8;
```